torch/_dynamo/compiled_autograd.py

```diff
@@ -1,6 +1,7 @@
 # mypy: allow-untyped-defs
```

Comment:
I also feel that the repro in https://github.com/pytorch/pytorch/issues/138920 isn't too long, and maybe we can add it as a unit test in this PR